packages/runtime-dom/src/directives/vModel.ts

```diff
@@ -85,7 +85,15 @@ export const vModelText: ModelDirective<
         return
       }
     }
-    el.value = value == null ? '' : value
+    const newValue = value == null ? '' : value
+    if (
+      el.minLength > 0 &&
+      el.value.length === newValue.length &&
```

Comment:
What's the point of checking `minLength` and `.length` here?
It seems we can simply check if they are equal.

Reply:
when value is a long string it can help prevent full comparison if `minLength` is not set or `length` are the same.